en/micro-integrator/docs/use-cases/tutorials/sending-a-simple-message-to-a-datasource.md

```diff
@@ -85,9 +87,19 @@ Follow the steps given below to create the data service file:
     </tr>
     </tbody>
     </table>
+    
+A data service file (DBS file) will now be created in your data service
+project as shown below.
 
-4.  Click **Next** and start adding the datasource connection details
-    given below.
+![](../../assets/img/tutorials/data_services/dataservice_view.png)
+
+##### Creating the datasource connection
+
+1.  Click **Data Sources** to expand the section.
+    ![](../../assets/img/tutorials/data_services/data_source_expanded.png)
+2.  Click **Add New** to open the **Create Datasource** page.
+    ![](../../assets/img/tutorials/data_services/add_data_source.png)
```

Comment:
@DMadhuranga - Shall we change this to an empty screenshot? The descriptions are more meaningful that way.